src/iotjs_binding.cpp

```diff
@@ -344,19 +299,17 @@ JResult JObject::Call(JObject& this_, JArgList& arg) {
     }
   }
 
-  bool is_ok = jerry_api_call_function(_obj_val.u.v_object,
-                                       this_obj_p,
-                                       &res,
-                                       val_args,
-                                       val_argv);
+  res = jerry_call_function(_obj_val, this_.raw_value(), val_args, val_argv);
 
   if (val_args) {
     delete [] val_args;
   }
 
-  JResultType type = is_ok ? JRESULT_OK : JRESULT_EXCEPTION;
+  JResultType type = jerry_value_has_error_flag(res)
+      ? JRESULT_EXCEPTION
+      : JRESULT_OK;
```

Comment:
There was some similar removed ? : code and they put ? and : in the same column.